config.php

```diff
@@ -12,11 +12,17 @@
   		die("Connection failed: " . $conn->connect_error);
 	}
 
-	$getpass = $_GET["passwords"];
+	session_start();
+
+	// initial login
+	if (isset($_GET["user"]) && isset($_GET["passwords"])) {
+		$getuser = $_GET["passwords"];
+		$getpass = $_GET["passwords"];
 		
-	$sql = "SELECT * FROM btglogin WHERE passwords = '$getpass'";
-	$result = $conn->query($sql);
-  		// output data of each row
+		// todo sanitize inputs
+		$sql = "SELECT * FROM btglogin WHERE user = '$getuser' AND passwords = '$getpass'";
```

Comment:
Guessing `user` isn't a column in this table, I'm getting:
```
Uncaught mysqli_sql_exception: Unknown column 'user' in 'where clause'
```

Reply:
whoops let me fix that